lib/archethic_web/api/graphql/schema/transaction_type.ex

```diff
@@ -58,7 +58,7 @@ defmodule ArchethicWeb.API.GraphQL.Schema.TransactionType do
     field(:code, :string)
     field(:content, :content)
     field(:ownerships, list_of(:ownership))
-    field(:recipients, list_of(:address))
+    field(:recipients, list_of(:recipient))
```

Comment:
Doing this means that every App that are using this graphql API will be broken when the release will be up on mainnet.
We should find a way to handle this update by versionning the API maybe

Reply:
Actually versioning on GraphQL API is not the most opiniated approach and was a first goal of GraphQL vs Rest approach' 

https://graphql.org/learn/best-practices/#versioning

Reply:
So this means we should have a new field like `namedActionRecipient` ?
This field would return the new format of recipient while the old field `recipients` could continue to return the old format of recipient (list of addresses)

Reply:
Can be an idea